.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] You did a great job on your `first project` but please always adhere to the `project guidelines`. As per project requirement, you need to create the `test. md` file and put the path of the file into the `.gitignore`. It will help you to understand how the `.gitignore` help us to ignore the unwanted file in your repo while staging.

- Please look at the below snippet in order to fulfil the project requirements.
![Screenshot from 2023-04-25 17-01-27](https://user-images.githubusercontent.com/66476329/234263715-8b128e42-877f-44db-8b1b-fc99ac737c3e.png)

Reply:
@skyv26 I did make the test.md file and added in .gitignore file but I forget to push it thank you

Reply:
I corrected that already